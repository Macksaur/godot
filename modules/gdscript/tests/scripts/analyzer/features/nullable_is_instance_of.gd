func test():
    var arr: Array? = null
    if is_instance_of(arr, TYPE_ARRAY):
        print(arr)
    else:
        print("arr is null")