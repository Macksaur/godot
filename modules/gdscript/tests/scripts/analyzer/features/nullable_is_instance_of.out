GDTEST_OK
arr is null
